components/service/glean/src/main/java/mozilla/components/service/glean/scheduler/GleanLifecycleObserver.kt

```diff
@@ -34,5 +37,6 @@ internal class GleanLifecycleObserver : LifecycleObserver {
     @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
     fun onEnterForeground() {
         Baseline.sessions.add()
+        Baseline.duration.start()
```

Comment:
I think this works to send the duration of the last foreground session *only* because we send the baseline ping on going to background and *only* because ping data collection happens synchronously.  Just wondering if we should document that dependency somehow / somewhere.

Reply:
Good point. I've added a comment above this line.